modules/cnf-measuring-latency.adoc

```diff
@@ -37,6 +37,11 @@ The tests introduce the following environment variables:
 |`LATENCY_TEST_RUNTIME`
 |Specifies the amount of time in seconds that the latency test must run. The default value is 300 seconds.
 
+[NOTE]
+====
+To prevent the Ginkgo 2.0 test suite from timing out before the latency tests complete, set the `-ginkgo.timeout` flag to a value greater than the `LATENCY_TEST_RUNTIME` value. If you also set a `LATENCY_TEST_DELAY` value then you must set `-ginkgo.timeout` to a value greater than the `LATENCY_TEST_RUNTIME` + `LATENCY_TEST_DELAY` + 2 minutes. The default timeout value for the Ginkgo 2.0 test suite is 1 hour.
```

Comment:
Small NIT:
```suggestion
To prevent the Ginkgo 2.0 test suite from timing out before the latency tests complete, set the `-ginkgo.timeout` flag to a value greater than the `LATENCY_TEST_RUNTIME`+ 2 minutes value. If you also set a `LATENCY_TEST_DELAY` value then you must set `-ginkgo.timeout` to a value greater than the `LATENCY_TEST_RUNTIME` + `LATENCY_TEST_DELAY` + 2 minutes. The default timeout value for the Ginkgo 2.0 test suite is 1 hour.
```